lib/rules/property-no-unknown/README.md

```diff
@@ -130,3 +130,25 @@ a {
   -moz-overflow-scrolling: center;
 }
 ```
+
+### `ignoreSelectors: ["/regex/", /regex/, "string"]`
+
+Some CSS extensions allow arbitrary child properties of certain selectors and pseudo-classes, which shouldn't be checked against a whitelist of known properties. One such example is [ICSS](https://github.com/css-modules/icss), which defines `:export` and `:import` pseudo-classes allowing any valid CSS property name to be nested in their declaration blocks.
```

Comment:
We use standard CSS syntax in our docs. Code related to methodologies or language extensions can confuse users unfamiliar with them. Also, methodologies like ICSS will come and go, but CSS will stick around. Using plain CSS in our examples has made our docs resilient over the past 5 years.

I think we can strip this paragraph out and just use something like `:root` in the example as `ignoreProperties` already shows an example of regex usage. For example:

---

### `ignoreSelectors: ["/regex/", /regex/, "string"]`

Given:

```js
[":root"]
```

The following patterns are *not* considered violations:

```css
:root {
  my-property: blue;
}
```

Reply:
Makes sense.